dev/com.ibm.ws.com.unboundid/resources/SunOne.ldif

```diff
@@ -30,19 +40,9 @@ objectClass: inetOrgPerson
 objectClass: person
 objectClass: top
 title: Test Persona
-userPassword:: e1NIQX1QMVlCMmhUK0lpbk8vMjlZaUltcFpKaGVwWUE9
```

Comment:
Did this need to be decoded? Probably doesn't really matter since it is easier to read this way.